models/Document/Editable/Area.php

```diff
@@ -79,12 +79,27 @@ public function admin(): void
         $this->outputEditmode('</div>');
     }
 
-    private function renderDialogBoxEditables(array $config, EditableRenderer $editableRenderer, string $dialogId): void
+    private function renderDialogBoxEditables(array|Model\Document\Editable $config, EditableRenderer $editableRenderer, string $dialogId): array
     {
+        if ($config instanceof BlockInterface || $config instanceof Area) {
+            // Unsupported element was passed (e.g., Block, Areablock, ...)
+            // or an Areas was passed, which is not supported to avoid too long editable names
+            $config = [];
+        } elseif ($config instanceof Model\Document\Editable) {
+            // Map editable to array config
+            $config = [
+                'type' => $config->getType(),
+                'name' => $config->getName(),
+                'label' => $config->getLabel(),
+                'config' => $config->getConfig(),
+                'description' => $config->getDialogDescription(),
+            ];
```

Comment:
What about default values? And other settings like `minValue`/`maxValue` for `Numeric` editables, etc.?